R/outbreak_setup.R

```diff
@@ -1,13 +1,9 @@
 #' Set up initial cases for branching process
 #' @author Joel Hellewell
 #'
-#' @param num.initial.cases Integer number of initial cases
-#' @param incfn function that samples from incubation period Weibull distribution; generated using dist_setup
-#' @param delayfn function that samples from the onset-to-hospitalisation delay Weibull distribution; generated using dist_setup
-#' @param k Numeric skew parameter for sampling the serial interval from the incubation period
-#' @param prop.asym Numeric proportion of cases that are sublinical (between 0 and 1)
+#' @inheritParams outbreak_model
 #'
-#' @return data.table of cases in outbreak so far
+#' @return `data.table` of cases in outbreak so far
```

Comment:
column names + types?

Reply:
Addressed in ba91591d7ca87fd8a3ec596d7d58131339de5d37.